build_tools/cmake/build_host_and_android.sh

```diff
@@ -1,88 +0,0 @@
-#!/bin/bash
-# Copyright 2020 The IREE Authors
-#
-# Licensed under the Apache License v2.0 with LLVM Exceptions.
-# See https://llvm.org/LICENSE.txt for license information.
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
-# Cross-compile the IREE project towards Android with CMake. Designed for CI,
-# but can be run manually. This uses previously cached build results and does
-# not clear build directories.
```

Comment:
This is still used by a BuildKite pipeline: https://github.com/iree-org/iree/blob/3186975420c9e0ca83db267709df1e81d2ce24a6/build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml#L7-L11

https://buildkite.com/iree/iree-test-android

https://github.com/iree-org/iree/issues/9855 "Android on-device test"

Reply:
Ahh.. sorry. revert the change for that script..